content/docs/command-reference/plots/index.md

```diff
@@ -98,72 +100,50 @@ names in the `train` array below:
 
 ## Plot templates (data series only)
 
-Users have the ability to change the way data-series plots are displayed by
-modifying the [Vega-Lite specification](https://vega.github.io/vega-lite/), thus
-generating plots in the style that best fits the their needs. This keeps
-<abbr>DVC projects</abbr> programming language agnostic, as it's independent
-from user display configuration and visualization code.
+DVC uses [Vega-Lite](https://vega.github.io/vega-lite/) JSON specifications to
+create visualizations out of users data. Pre-defined specifications are called
+templates.
 
-Built-in _plot templates_ are stored in the `.dvc/plots/` directory. The default
-one is called `default.json`. It can be changed with the `--template` (`-t`)
-option of `dvc plots show` and `dvc plots diff`. For templates in the
-`.dvc/plots/` directory, the path and the json extension are not required: you
-can specify only the base name e.g. `--template scatter`.
+DVC has few built in templates. They are stored within
+[`dvc-render`](https://github.com/iterative/dvc-render) library. The default one
+is called `linear`. It can be changed with the `--template` (`-t`) option of
+`dvc plots show` and `dvc plots diff`. The argument provided to `--template`
+option can be either a built-in template name or path to a JSON specification.
+For templates stored in the `.dvc/plots/` directory, the path and the json
+extension are not required: user can specify only the base name e.g.
+`--template scatter`.
 
 DVC has the following built-in plot templates:
 
-- `default` - linear plot
+- `linear` - linear plot, used by default
 - `scatter` - scatter plot
 - `smooth` - linear plot with LOESS smoothing, see
   [example](/doc/command-reference/plots#example-smooth-plot)
 - `confusion` - confusion matrix, see
   [example](/doc/command-reference/plots#example-confusion-matrix)
+- `confusion_normalized` - confusion matrix with values normalized to <0, 1>
+  range
+- `simple` - simplest linear template, good for learning how to work with
+  [`templates`](/doc/command-reference/plots/templates) command
```

Comment:
Isn't it more useful as a baseline for creating custom templates?

Reply:
What's the difference with `linear`?

Reply:
Main difference is lack of interactivity in simple. It also has like 4 times less lines, so its easier to comprehend. Moved up.